### DISCUSSION TITLE
How to listen to database event

### DISCUSSION THREAD

**Discussion by easy-Coder** [serverpod/serverpod#620] [Q&A] [answered]
Hello, I want to open a connection to client and push message if an insertion was made to a table in the database.

**Comment by vlidholt** [comment, marked as answer]
Hi! You probably want to use Serverpods message central for this.

Here is some documentation:
https://docs.serverpod.dev/concepts/streams

But you can also check out the source code for Pixorama here:
https://pixorama.live/
